docs/uml/macros.iuml

```diff
@@ -381,7 +379,7 @@ skinparam defaultMonospacedFontName Andale Mono
 !endif
 
 !ifdef SHOW_CONTRACT && SHOW_CONSTRUCTOR
-  Constructor "1" o--- "0..1" Contract
+  Constructor "1" ---o "0..1" Contract
```

Comment:
I think we can get rid of the "0..1" here, or maybe move it to the left side (since not all Contracts will have Constructors?)